### ISSUE TITLE
' Uncaught TypeError: (row[columnKey] || "").toLowerCase is not a function '  When sorting or filtering

### ISSUE THREAD

**Issue by moejarv** [MyPureCloud/fixtable-ember#70]
Happens to me on latest chrome and firefox, when I am trying to sort or filter records.

Here is my setup:

A simple line in template:
`{{fixtable-grid columns=fixTableCols content=reports filterDebounce=1000 }}`

The column definition looks like this: 
```
  fixTableCols: [
    {key:'name', header:'Name',  sortable:true, filter:{type:'search'}, component:'fixtable-name-link'},
    {key:'label', header:'Label',sortable:true, filter:{type:'search'} },
    {key:'description', header:'Description',sortable:true,filter:{type:'search'} },
    {key:'reportPath', header:'Path',sortable:true, filter:{type:'search'} },
    {key:'reportSections', header:'Sections',sortable:true,  component:'report-fixtable-section'},
    {key:'parameters', header:'Parameters',sortable:true,  component:'report-fixtable-parameter'},
  ],

```

and the model looks like this:
```
  name: attr('string'),
  label: attr('string'),
  description: attr('string'),
  reportPath: attr('string'),
  // relations
  reportSections: DS.hasMany('report-section'),
  parameters: DS.hasMany('parameter'),
```

I have found out that if I change the sort / filter key part of the code from simple [key] lookup to .get(), the sort and filter works. 
(addon/componments/fixtable-grid.js)
`Ln 162:   var cellData = (row[columnKey] || '').toLowerCase(); `
change to 
`Ln 162:   var cellData = (row.get(columnKey) || '').toLowerCase(); `

Could this be due to I have hasMany in the model? 
Or it is something else on my setup is quirky?

And, is this the right way to fix this? If yes, I can send in pull request : )

**Comment by menewman** [comment]
What version of this addon are you using? This seems like the issue that was fixed by PR #58, so I believe updating your version will make this error go away.

**Comment by moejarv** [comment]
Ahh.. My bad! I am still on 0.2.1.. 

Closing issue..
